pkg/volume/azure_dd/attacher.go

```diff
@@ -214,7 +216,11 @@ func (attacher *azureDiskAttacher) MountDevice(spec *volume.Spec, devicePath str
 
 	if err != nil {
 		if os.IsNotExist(err) {
-			if err := os.MkdirAll(deviceMountPath, 0750); err != nil {
+			dir := deviceMountPath
+			if runtime.GOOS == "windows" {
+				dir = filepath.Dir(deviceMountPath)
```

Comment:
In which case will it be different? Better add a comment here.

Reply:
done:
// in windows, as we use mklink, only need to MkdirAll for parent directory